tests/util/models/keras/models.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2021 The Trieste Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Utilities for creating (Keras) neural network models to be used in the tests.
```

Comment:
I was expecting this file to contain instances of the models, similar to gpflow / gpflux. Does it not apply here?

Reply:
I needed utility functions for creating neural nets and models, files for these utlities should reflect trieste repo structure I think - I can rename the file name to something else?